### ISSUE TITLE
NTR: Mosaic retinal pigmentation in mother (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7986]
New term request Mosaic retinal pigmentation in mother (elthat_120827213341): Maternal; due to X-linked inactivation

**Comment by pnrobinson** [comment]
@MickeySegal
This term is potentially confusing since I believe different communities might interpret this as a genetic mosaic or otherwise, see
https://www.cambridge.org/core/books/abs/retinal-development/retinal-mosaics/31B667AC2CA26B58387E8AE786389A2C
Could you provide more context as to the intended use case?

**Comment by MickeySegal** [comment]
Our term is currently "Retinal mottling in mother" and we are using https://hpo.jax.org/app/browse/term/HP:0007814 even though that is the finding in a patient.  

The context is the disease "Ocular albinism, X-linked males", where the finding in the mother identifies the heterozygote state.

**Comment by pnrobinson** [comment]
I do not have access to the above chapter. Is this something that is offered as a clinicla test or is it done occasionally in research settings? I understand the application but have never heard of this being done in a clinical context.

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1232/
"The correct diagnosis becomes clear when the ophthalmologist dilates the pupils of the eyes of the mother of a male child and finds the classic mosaic retinal pigmentation of the X-linked carrier state."

**Comment by pnrobinson** [comment]
Thanks, but this is not a good definition because it is circular!
"classic mosaic retinal pigmentation" -- if you do not know what this is, the definition is not helpful.

**Comment by MickeySegal** [comment]
That's why we switched to "retinal mottling".  

This reminds me of Samuel Johnson's definition: 
https://www.goodreads.com/quotes/258751-network-any-thing-reticulated-or-decussated-at-equal-distances-with
“Network: Any thing reticulated or decussated, at equal distances, with interstices between the intersections [....]  
Reticulated: Made of network; formed with interstitial vacuities.”

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001041
- label: Maternal retinal mottling
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: PMID:17460294
- def: Mottling (spots or blotches with different shades) of the retinal pigment epithelium in the mother of the index patient. Reintal mottling may indicate a mosaic state in the mother.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7986
- synonyms: 


(Note, we will be rearranging the maternal history terms)

**Comment by MickeySegal** [comment]
Mosaic also has another meaning in genetics, referring to two entirely different cell lines.  It might be better instead to use "X-liked inactivation".

**Comment by pnrobinson** [comment]
check this

**Comment by pnrobinson** [comment]
Mottling is what we observe, I do not think there is a lot of potential for confusin with cell lines.
I updated the definition to this:
Mottling (spots or blotches with different shades) of the retinal pigment epithelium in the mother of the index patient. Retinal mottling may indicate a mosaic X-linked inactivation state in the mother.
